### PR TITLE
Fixes #151 - client install first iteration

### LINKED ISSUE #151: Use ansible for new client machine install.
To replace @jlaforga script.

### PR DESCRIPTION
**Pull request by ugocupcic**
This is the first iteration of setting up a production machine using ansible. It still needs to be tested with a real hand, the icons / scripts need to be cleaned up with a real hand as well. 

Additional steps can then be done (such as running the tests when building for example).

### CODE REVIEW

File: ansible/roles/desktop_setup/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Install pycurl
+  apt: name={{item}}
+  with_items:
+    - python-pycurl
```
Comment (AndriyPt) [inline ansible/roles/desktop_setup/tasks/main.yml]: you might use here simple version of apt module call without loop "apt: name=python-pycurl state=present"

File: ansible/roles/desktop_setup/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- name: Add PyCharm ppa
+  apt_repository: repo='ppa:mystic-mirage/pycharm'
+
+- name: Update package lists
+  apt: update_cache=yes
+
+- name: Install programming tools
+  apt: name={{item}}
+  with_items:
+    - pycharm-community
```
Comment (AndriyPt) [inline ansible/roles/desktop_setup/tasks/main.yml]: the same applies here about simpler form of apt module usage without loop

File: data/shadow_robot-indigo.rosinstall [hunk 3]
```diff
@@ -28,10 +28,10 @@
...
     local-name: ethercat_grant
 
-- git:
-    local-name: sr_contrib
-    uri: 'https://github.com/shadow-robot/sr-contrib.git'
-    version: 'indigo-devel'
+# - git:
+#     local-name: sr_contrib
+#     uri: 'https://github.com/shadow-robot/sr-contrib.git'
+#     version: 'indigo-devel'
```
Comment (AndriyPt) [inline data/shadow_robot-indigo.rosinstall]: cleanup candidate

Comment (ugocupcic) [inline data/shadow_robot-indigo.rosinstall]: https://github.com/shadow-robot/sr-build-tools/issues/155

**Pull request merged**